src/lib/components/pages/products/product-pricing-pannel.svelte

```diff
@@ -158,7 +165,7 @@
 			<span class="text-sm mr-1">
 				{userShippingAddress}
 			</span>
-			<IconButton icon={MapPin} rounded size="xs" variant="light" />
+			<IconButton icon={MapPin} rounded size="xs" variant="light" disabled={$isLoadingStore}/>
```

Comment:
read requirements again, the button map pin is not related here, we only care about plus/minus/add to cart/ variant select buttons